st2actions/st2actions/runners/ssh/paramiko_ssh.py

```diff
@@ -349,6 +349,7 @@ def run(self, cmd, timeout=None, quote=False):
         chan = transport.open_session()
 
         start_time = time.time()
+        chan.get_pty()
```

Comment:
I'll check tomorrow what fabric does but without this running apt-get commands fail with return code -1 and no visible error. http://serverfault.com/questions/489939/ssh-session-never-closes-when-executing-apt-get-install recommends allocating a pseudo tty. With this, puppet_bootstrap always succeeds.